docs/dev_guide/diagnostic_ref/dpct1039.rst

```diff
@@ -31,4 +31,52 @@ Suggestions to Fix
 
 If the first parameter of an atomic function points to a local memory address
 space, replace the atomic function name with an atomic function name that
-includes the template parameters.
\ No newline at end of file
+includes the template parameters.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+    __device__ void d(int* s) {
+      atomicAdd(s, 1);
+    }
+    
+    __global__ void k() {
+      __shared__ int shared;
+      d(&shared);
+    }
+
+results in the following migrated SYCL code:

```

Comment:
Add "*" to SYCL - not yet attributed on page. Need to attribute first use only.

Reply:
done

Reply:
done